src/App.js

```diff
@@ -573,11 +574,10 @@ function App() {
           </Grid>
         </Grid>
       </Fade>
-      <div className={`${classes.logo} ${logoLoaded?classes.logoLoaded:""}`}>
-        <a href="https://greenstand.org/" target="_blank" rel="noopener noreferrer">
-          <img alt="logo" src={logoSrc} />
-        </a>
-      </div>
+      <Logo
+        logoLoaded={logoLoaded}
+        logoSrc={logoSrc}
+      />
```

Comment:
Just for discussion, how about putting the load process into the Logo, I don't if this idea has any problem, just thinking, if you look at the logo as a component/class, then the way to load the logo is might be an appropriate thing this class should KNOW.

Reply:
I think it makes a lot of sense for the logo component to own that logic!  I'll adjust the PR

Reply:
After looking at this more, I think keeping the logo loading logic outside of the component leaves the component with a single responsibility and superior testability.  Let me know if you agree and I can merge